model_templates/README.md

```diff
@@ -0,0 +1,32 @@
+# Custom Inference Model Templates
+This folder provides templates for building and deploying custom models in DataRobot. Use the templates as an example structure for your own custom models.
+
+### DataRobot User Model Runner
+The examples in this repository use the DataRobot User Model Runner (DRUM).  For more information on how to use and write models with DRUM, reference the [readme](./custom_model_runner/README.md).
+
+### Sample Models
+The [model_templates](model_templates) folder contains sample models that work with the provided template environments. For more information about each model, reference the readme for every example:
+
+##### Inference Models
+* [Scikit-Learn sample model](model_templates/python3_sklearn)
+* [Scikit-Learn sample unsupervised anomaly detection model](model_templates/python3_sklearn_anomaly)
+* [PyTorch sample model](model_templates/python3_pytorch)
+* [XGBoost sample model](model_templates/python3_xgboost)
+* [Keras sample model](model_templates/python3_keras)
+* [Keras sample model + Joblib artifact](model_templates/python3_keras_joblib)
+* [PyPMML sample model](model_templates/python3_pmml)
+* [R sample model](model_templates/r_lang)
+* [Java sample model](model_templates/java_codegen)
+* [Julia sample models](model_templates/julia)
+
+##### Training Models
```

Comment:
Let's remove the training models section from here.

Reply:
done